<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Credit_calc</class>
 <widget class="QDialog" name="Credit_calc">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1080</width>
    <height>330</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>3</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>1080</width>
    <height>330</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1080</width>
    <height>330</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Credit calc</string>
  </property>
  <widget class="QWidget" name="gridLayoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>1068</width>
     <height>317</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="1" column="2">
     <widget class="QSpinBox" name="credit_time">
      <property name="maximumSize">
       <size>
        <width>300</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">
     background-color: rgb(138, 203, 192);
     border-style: outset;
     border-width: 5px;
     border-radius: 20px;
     border-color: beige;
     font: bold 20px;
     padding: 6px;</string>
      </property>
      <property name="minimum">
       <number>1</number>
      </property>
     </widget>
    </item>
    <item row="0" column="3">
     <widget class="QLabel" name="label_5">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>0</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>300</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">
     background-color: rgb(138, 203, 192);
     border-style: outset;
     border-width: 5px;
     border-radius: 20px;
     border-color: beige;
     font: bold 20px;
     padding: 6px;</string>
      </property>
      <property name="text">
       <string>Ежемесячный платёж</string>
      </property>
     </widget>
    </item>
    <item row="1" column="3">
     <widget class="QLabel" name="label_6">
      <property name="styleSheet">
       <string notr="true">
     background-color: rgb(138, 203, 192);
     border-style: outset;
     border-width: 5px;
     border-radius: 20px;
     border-color: beige;
     font: bold 20px;
     padding: 6px;</string>
      </property>
      <property name="text">
       <string>Переплата по кредиту</string>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="label_3">
      <property name="maximumSize">
       <size>
        <width>230</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">
     background-color: rgb(138, 203, 192);
     border-style: outset;
     border-width: 5px;
     border-radius: 20px;
     border-color: beige;
     font: bold 20px;
     padding: 6px;</string>
      </property>
      <property name="text">
       <string>Процентная ставка</string>
      </property>
     </widget>
    </item>
    <item row="3" column="2">
     <widget class="QRadioButton" name="type_annual">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>60</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>300</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">
     background-color: rgb(138, 203, 192);
     border-style: outset;
     border-width: 5px;
     border-radius: 20px;
     border-color: beige;
     font: bold 20px;
     padding: 6px;</string>
      </property>
      <property name="text">
       <string>Аннуитетный</string>
      </property>
     </widget>
    </item>
    <item row="3" column="0" rowspan="2">
     <widget class="QLabel" name="label_4">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>130</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>230</width>
        <height>130</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">
     background-color: rgb(138, 203, 192);
     border-style: outset;
     border-width: 5px;
     border-radius: 20px;
     border-color: beige;
     font: bold 20px;
     padding: 6px;</string>
      </property>
      <property name="text">
       <string>Типа кредита</string>
      </property>
     </widget>
    </item>
    <item row="2" column="3">
     <widget class="QLabel" name="label_7">
      <property name="styleSheet">
       <string notr="true">
     background-color: rgb(138, 203, 192);
     border-style: outset;
     border-width: 5px;
     border-radius: 20px;
     border-color: beige;
     font: bold 20px;
     padding: 6px;</string>
      </property>
      <property name="text">
       <string>Общая выплата по кредиту</string>
      </property>
     </widget>
    </item>
    <item row="4" column="2">
     <widget class="QRadioButton" name="type_diff">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>60</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>300</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">
     background-color: rgb(138, 203, 192);
     border-style: outset;
     border-width: 5px;
     border-radius: 20px;
     border-color: beige;
     font: bold 20px;
     padding: 6px;</string>
      </property>
      <property name="text">
       <string>Дифференциированный</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="label_2">
      <property name="maximumSize">
       <size>
        <width>230</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">
     background-color: rgb(138, 203, 192);
     border-style: outset;
     border-width: 5px;
     border-radius: 20px;
     border-color: beige;
     font: bold 20px;
     padding: 6px;</string>
      </property>
      <property name="text">
       <string>Срок кредита (лет)</string>
      </property>
     </widget>
    </item>
    <item row="0" column="0">
     <widget class="QLabel" name="label">
      <property name="maximumSize">
       <size>
        <width>230</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">
     background-color: rgb(138, 203, 192);
     border-style: outset;
     border-width: 5px;
     border-radius: 20px;
     border-color: beige;
     font: bold 20px;
     padding: 6px;</string>
      </property>
      <property name="text">
       <string>Сумма кредита</string>
      </property>
     </widget>
    </item>
    <item row="0" column="4">
     <widget class="QLabel" name="month_payment">
      <property name="minimumSize">
       <size>
        <width>250</width>
        <height>0</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">
     background-color: rgb(138, 203, 192);
     border-style: outset;
     border-width: 5px;
     border-radius: 20px;
     border-color: beige;
     font: bold 20px;
     padding: 6px;</string>
      </property>
      <property name="text">
       <string>1</string>
      </property>
     </widget>
    </item>
    <item row="1" column="4">
     <widget class="QLabel" name="pereplata">
      <property name="minimumSize">
       <size>
        <width>250</width>
        <height>0</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">
     background-color: rgb(138, 203, 192);
     border-style: outset;
     border-width: 5px;
     border-radius: 20px;
     border-color: beige;
     font: bold 20px;
     padding: 6px;</string>
      </property>
      <property name="text">
       <string>1</string>
      </property>
     </widget>
    </item>
    <item row="2" column="4">
     <widget class="QLabel" name="all_payings">
      <property name="minimumSize">
       <size>
        <width>250</width>
        <height>0</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>250</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">
     background-color: rgb(138, 203, 192);
     border-style: outset;
     border-width: 5px;
     border-radius: 20px;
     border-color: beige;
     font: bold 20px;
     padding: 6px;</string>
      </property>
      <property name="text">
       <string>1</string>
      </property>
     </widget>
    </item>
    <item row="3" column="3" rowspan="2" colspan="2">
     <widget class="QPushButton" name="Calc">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>130</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>600</width>
        <height>130</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">
     background-color: rgb(138, 203, 192);
     border-style: outset;
     border-width: 5px;
     border-radius: 20px;
     border-color: beige;
     font: bold 40px;
     padding: 6px;</string>
      </property>
      <property name="text">
       <string>Расчитать</string>
      </property>
     </widget>
    </item>
    <item row="0" column="2">
     <widget class="QSpinBox" name="credit_sum">
      <property name="maximumSize">
       <size>
        <width>300</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">
     background-color: rgb(138, 203, 192);
     border-style: outset;
     border-width: 5px;
     border-radius: 20px;
     border-color: beige;
     font: bold 20px;
     padding: 6px;</string>
      </property>
      <property name="minimum">
       <number>1</number>
      </property>
      <property name="maximum">
       <number>99999999</number>
      </property>
     </widget>
    </item>
    <item row="2" column="2">
     <widget class="QSpinBox" name="credit_percent">
      <property name="maximumSize">
       <size>
        <width>300</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">
     background-color: rgb(138, 203, 192);
     border-style: outset;
     border-width: 5px;
     border-radius: 20px;
     border-color: beige;
     font: bold 20px;
     padding: 6px;</string>
      </property>
      <property name="minimum">
       <number>1</number>
      </property>
      <property name="maximum">
       <number>365</number>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
